kallisto/urls.py

```diff
@@ -7,6 +7,7 @@
 urlpatterns = patterns(
     '',
     url(r'^$', 'apps.homepage.views.homepage', name='homepage'),
+    url(r'^help$', 'apps.homepage.views.help', name='help'),
```

Comment:
I can't explain why, but I would have done `TemplateView.as_view(template_name='homepage/help.html')` or similar here. (Don't bother changing it, though.)